src/functions/template-tags/link.php

```diff
@@ -339,14 +339,16 @@ function tribe_get_listview_next_link( $term = null ) {
 }
 
 /**
- * Single Event Link
+ * Retrieves the link to a single event.
  *
- * Get link to a single event
+ * This function returns the URL to a single event post, or if the `$full_link` parameter is set to true,
+ * it outputs a complete HTML `<a>` tag with the event title as the link text. If no event is found, it returns false.
  *
- * @param WP_Post|int $post_id   Optional. WP Post that this affects
- * @param bool        $full_link Optional. If true outputs a complete HTML <a> link, otherwise only the URL is output
+ * @since 2.0.1
+ * @param WP_Post|int|null $post_id   Optional. The event post ID or WP_Post object. Defaults to the current post if not provided.
+ * @param bool             $full_link Optional. If true, outputs a complete HTML `<a>` tag. Defaults to false, returning just the URL.
```

Comment:
Let's add a line of space here:
```suggestion
 * @since 2.0.1
 *
 * @param WP_Post|int|null $post_id   Optional. The event post ID or WP_Post object. Defaults to the current post if not provided.
 * @param bool             $full_link Optional. If true, outputs a complete HTML `<a>` tag. Defaults to false, returning just the URL.
```